src/vs/workbench/contrib/notebook/common/notebookCommon.ts

```diff
@@ -36,6 +36,15 @@ export enum CellOutputKind {
 	Rich = 3
 }
 
+export enum KernelsLanguage {
```

Comment:
It would be great if you could follow up and find the places that we have these values hardcoded already, such as https://github.com/microsoft/azuredatastudio/blob/main/src/sql/workbench/services/notebook/browser/models/notebookModel.ts#L800 and https://github.com/microsoft/azuredatastudio/blob/main/extensions/notebook/src/common/notebookUtils.ts#L127 and then replace them with usages of this enum.